packer/scylla.json

```diff
@@ -164,6 +164,13 @@
     }
   ],
   "provisioners": [
+    {
+      "type": "shell",
+      "inline": [
+        "if [ {{build_name}} = aws -o {{build_name}} = azure ]; then sudo apt-get update; sudo DEBIAN_FRONTEND=noninteractive apt purge -y linux-aws* linux-headers-{{build_name}}* linux-image*{{build_name}}* linux-modules*-{{build_name}}*; sudo DEBIAN_FRONTEND=noninteractive apt-get install -y linux-{{build_name}}-lts-22.04; sudo reboot ; fi"
```

Comment:
Should 'sudo apt-get update' be 'sudo apt-get update -y' ?

Reply:
Also, with DEBIAN_FRONTEND=noninteractive ?

Reply:
> Should 'sudo apt-get update' be 'sudo apt-get update -y' ?

`apt-get update` is good. later under `scylla_install_image` we are doing the upgrade. This is only so we can install new packages

Reply:
> Also, with DEBIAN_FRONTEND=noninteractive ?

Yes, when we removing kernel, there is another user input that require, since we are running during the build process we need to define this

Reply:
> > Should 'sudo apt-get update' be 'sudo apt-get update -y' ?
> 
> `apt-get update` is good. later under `scylla_install_image` we are doing the upgrade. This is only so we can install new packages

I'm missing the '-y' here, that's all.

Reply:
oh, `apt-get update` doesn't require `-y`